src/api/wix/WixToolset.Data/WixStandardLibraryIdentifiers.cs

```diff
@@ -16,5 +16,10 @@ public static class WixStandardLibraryIdentifiers
         /// WiX Standard references for modules.
         /// </summary>
         public static readonly string WixStandardModuleReferences = "WixStandardModuleReferences";
+
+        /// <summary>
+        /// Default feature name.
+        /// </summary>
+        public static readonly string DefaultFeatureName = "WixDefaultFeature";
```

Comment:
Since Features sometimes have to be set from the command-line (like in `ADDLOCAL`) should we pick something more generic like "Main" or better?

Reply:
There's only the one feature and it's required, with no conditions or anything, so I didn't see any reason to even document it. What did I miss to force me to write doc? :)

Reply:
I guess the answer is for a setup dev to define a custom feature if they want to control the name. I was thinking if we used something generic like `MainFeature` that would be generally usable.

Reply:
Unless you'd like something like `DefaultFeatureName` attribute on `Package`? :)